bundles/org.eclipse.swt/Eclipse SWT WebKit/gtk/org/eclipse/swt/browser/WebKit.java

```diff
@@ -795,7 +797,7 @@ public void create (Composite parent, int style) {
 				break;
 			}
 			case SWT.KeyDown: {
-				if (!disableBrowserSearch && event.keyCode == 'f' && (event.stateMask & SWT.CTRL) == SWT.CTRL) {
+				if (!disableBrowserSearch && enableSearch && event.keyCode == 'f' && (event.stateMask & SWT.CTRL) == SWT.CTRL) {
```

Comment:
`disableBrowserSearch && enableSearch` is funny.
I would propose to rename `disableBrowserSearch` to `disableBrowserSearchGlobally` or something like that.